src/redisearch_api.c

```diff
@@ -25,6 +25,10 @@ IndexSpec* RS_CreateIndex(const char* name, RSGetValueCallback getValue, void* g
   return spec;
 }
 
+void RS_DropIndex(IndexSpec* sp) {
+  IndexSpec_Free(sp);
```

Comment:
I do not think the IndexSpec_Free actually free the dict of terms on the spec (the one that was added to get the inverted index out of the keyspace).